docker/verify/Jenkinsfile

```diff
@@ -347,14 +356,20 @@ node("docker-daemon") {
       parallel (
         "app": {
           dir("app") {
-            appHash = sh(script: 'git describe --always --tags --abbrev=7 HEAD', returnStdout: true).trim()
+            appHash = sh(script: "git tag -l '${escapedReleaseBranch}*' | tail -1", returnStdout: true).trim()
```

Comment:
if you want the latest tag on a current commit you could use `git describe --exact-match --abbrev=0`

Reply:
this won't work for the config, but I changed for something similar